test/museek/test_data_element.py

```diff
@@ -342,3 +342,29 @@ def test_flagged_std(self):
                              [54., 81., 54.],
                              [54., 54., 20.25]]) ** (1 / 2)
         np.testing.assert_array_equal(expect, std.squeeze)
+
+        def test_flagged_min(self):
```

Comment:
the added tests are not functional (missing indentation) and also not all new methods are covered